net.audiorelay.AudioRelay.metainfo.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.audiorelay.AudioRelay</id>
+  
+  <name>AudioRelay</name>
+  <summary>Stream audio between your devices</summary>
+  <developer_name>Asapha Halifa</developer_name>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
Is this redistributable? If there are issues with ToS, it should use extra-data I think.

Reply:
I think I can assume we can redistribute that, they don't even have any ToS on their website or in the app.

Reply:
> I think I can assume we can redistribute that

Actually, unless specifically stated, we can assume it cannot be redistributed.